.gitignore

```diff
@@ -0,0 +1 @@
+myenv/
```

Comment:
This is not necessary here and does not relate in a way with the PR title. Please, revert this change. Besides, project-specific gitignore files should mention things that are related to said project, not one's personal preference of naming virtualenvs. People tend to use `env`, `.env`, `venv`, `.venv` and other names for their virtualenvs but it doesn't make sense to enumerate every possible variation any human could come up with. Also, some people manage virtualenvs outside of the project dirs. For example, I use `pyenv-virtualenv` that puts them into the pyenv directory structure in the userspace.

Reply:
This change has been reverted